device-registry/src/main/scala/org/genivi/sota/device_registry/DeviceRepository.scala

```diff
@@ -75,9 +75,29 @@ object Devices {
     def pk = primaryKey("id", id)
   }
 
+  type GroupInfoType = Json
+  case class GroupInfo(groupName: String, namespace: Namespace, groupInfo: GroupInfoType)
+
+  class GroupInfoTable(tag: Tag) extends Table[GroupInfo] (tag, "DeviceGroup") {
+    def groupName = column[String]("group_name")
+    def namespace = column[Namespace]("namespace")
+    def groupInfo = column[Json]("group_info")
+
+    implicit val jsonColumnType = MappedColumnType.base[Json, String](
```

Comment:
I think we already have something similar for SystemInfo? we should abstract and reuse it